cmd/open.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/CircleCI-Public/circleci-cli/git"
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+)
+
+func projectUrl(remote *git.Remote) string {
+	return fmt.Sprintf("https://app.circleci.com/%s/%s/%s/pipelines",
```

Comment:
does go have a url templating function? I suspect these values should be safe given that they came from the git URL - but we should probably be using some equivalent of `encodeURIComponent` on these path components when assembling a URL